src/components/Upload/Upload.svelte

```diff
@@ -36,6 +38,10 @@
 		files = [];
 		dispatch('files', { files });
 	}
+
+	function onReject() {
+		toast.error({ msg: `Unaccepted file-types`, timeout: 4000, pos: 'top_right' });
```

Comment:
@Valexr how to define allowed file types?

Reply:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file#additional_attributes